Reworked the observability onboarding (#1)
Orenico10/ObservabilityOnboarding: learning/observability/part4.md

# Part 4: Observability platforms and APM

## Overview

In this part, you'll learn about insights Observability gives us.
Observability platforms give us a lot of insight and capabilities about our system from the telemetry data.

## Goals

- Learn about different Observability platforms.
- Understand how we can choose the right solution based on our system.
- Learn some more Observability concepts.

> ### Action Items
> 
> 1. Compare between SLA, SLO & SLI and how Observability helps with either of them
>    1. Explain in detail the importance of each
> 2. What is APM
> 3. Explain the differences between Synthetic monitoring & RUM, where they differ and not
> 4. Synthetic monitoring use case and pros
>    1. List three implementations of synthetic monitoring
> 5. List RUM features, choose 2 you like and explain what is their usage and how they takes place in real-world
> 6. List 2 more insights Observability gives your system and explain them
> 

Reviewer: ```suggestion
```

Reviewer: this is doubled with the outcome part of the doc

## Outcome

The outcome of this part is a detailed markdown explaining the concepts you have learned in this part. Your markdown should also contain comparison between different Observability platforms.

In your markdown make sure there is an answer the following questions:

> 

Reviewer: I would like to familiarize them with the idea of the 4 golden signals when facing to the user (https://www.dynatrace.com/knowledge-base/golden-signals/) because we talk in here more about things like sli and slo's and so on

1. Explain the difference between SLI, SLO, SLA, their relation and how observability helps with either of them
	- explain in depth the importance of each
2. What are KPIs? How do they relate to SLI, SLO and SLA?
3. Explain the differences between Synthetic monitoring & RUM, where they differ and not

> 4. Choose 2 RUM features you like and explain their usage and how they take place in the real-world.

Reviewer: what do you mean by that?

Author: Features that RUM usually has, the goal is that they talk about 2 features of RUM and explain them.

5. What is MTTD and MTTR?
6. What are alerts, why are they important? What is an alert manager?
7. List 2 more insights Observability gives your system and explain them

> 8. Provide a list of at least 3-4 Observability solutions today (also add a comparison between them).

Reviewer: ```suggestion
8. Provide a list of at least 3-4 Observability solutions today with comparison.
```
